test/modules/sungeun/init/printModuleInitOrder.good

```diff
@@ -1,22 +1,18 @@
 Initializing Modules:
    ChapelStandard
-    CPtr
-     SysBasic
-      SysCTypes
-     SysError
-     HaltWrappers
     CString
     Bytes
      ByteBufferHelpers
      BytesStringCommon
       String
+       SysCTypes
        StringCasts
+       SysBasic
     OwnedObject
-    ChapelDebugPrint
+     HaltWrappers
     ChapelBase
-    MemConsistency
     Atomics
-    NetworkAtomics
```

Comment:
Any idea why NetworkAtomics is no longer included (no big deal, just checking if this was correctly updated)

Reply:
Huh, good question...  I'll double check.

Reply:
Re-testing, it passes, but I don't have a theory for why it would not print now when it did before...  (and had believed it had just been reordered when I was looking at the diffs in the PR).  I think I'll have to dig into the generated code to understand what's happening on this one...